### ISSUE TITLE
"Enter" and "Installed" pings are wrongly sent on each browser restart on all branches

### ISSUE THREAD

**Issue by Softvision-PatriciuPop** [mozilla/webcompat-blipz-experiment#139]
[Affected versions]:
- Firefox Nightly Version 65.0a1, Build ID 20181029230149
- Firefox Unbranded Beta Version 63.0b12, Build ID 20181004174654
- Firefox Unbranded Release Version 62.0.3, Build ID 20181001155545

[Affected Platforms]:
- All Windows
- All Linux
- All Mac

[Prerequisites]:
- The "extensions.legacy.enabled" pref is set to "true" in about:config.
- The "xpinstall.signatures.required" pref is set to "false" in about:config.
- The "extensions.webcompat-blipz-experiment.variation" pref is set to "more-sentiment" in about:config.
- The webcompat_blipz_experiment-2.0.0 addon is installed.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Restart the browser and navigate to about:telemetry.
3. Observe the archived pings.

[Expected result]:
- No pings are sent after a restart.

[Actual result]:
- The "enter" and "installed" pings are wrongly sent.

[Notes]:
- This issue is reproducible on the other 3 branches of the study.
- I have attached a screen recording of the issue [here](https://drive.google.com/open?id=1FiHk1Pn9Mr7h8-BB_9azeA0UmRnV5PdC).

**Comment by wisniewskit** [comment]
Yes, unfortunately this isn't something we can control, it's just how shield experiments work. (Previously reported in https://github.com/mozilla/webcompat-blipz-experiment/issues/120)

**Comment by Softvision-PatriciuPop** [comment]
> 
> 
> Yes, unfortunately this isn't something we can control, it's just how shield experiments work. (Previously reported in #120)

To the best of my knowledge, no other studies our team tested have had this behavior. The "Enter" and "Installed" pings were sent only once when the add-on was installed. 
Leaving this behavior as it is could lead to issues with Normandy when reaching a certain number of "installed" pings that could lead to a smaller number of participants, and overall an incorrect sample of data.

**Comment by wisniewskit** [comment]
Thanks for letting me know! I'll investigate this as I can. (But this is still a duplicate of #120).
